Python/Python.sublime-syntax

```diff
@@ -1056,7 +1056,7 @@ contexts:
               captures:
                 1: variable.parameter.class-inheritance.python
                 2: keyword.operator.assignment.python
-            - match: (?={{path}})
+            - match: (?=(?!None|False|True|Ellipsis|NotImplemented|__debug__){{path}})
```

Comment:
I feel like this should use a variable or something ideally

Reply:
An alternative might be to include `constants` context before this pattern if we are concerned about consistency in case of more of them being added in future.

As all those constants are matched in separate patterns and this negative lookahead seems just required in this special case, I wonder what value a putting them into a variable has.

In order to ensure not any of those special constants is otherwise matched the negative lookahead would probably be needed in front of `{{path}}` or even `{{identifier}}` but I am concerned about too many other implications.

Reply:
... which arises a general design topic. What I have in mind is just the way how predefined keywords are prevented from being scoped as identifier in Java. Such a change - if required - would have some larger impact, I guess.

Reply:
> if we are concerned about consistency in case of more of them being added in future.

that was indeed my worry :)